### ISSUE TITLE
No telemetry message on first run

### ISSUE THREAD

**Issue by millette** [gatsbyjs/gatsby#12871]
Just upgraded an existing project to Gatsby 2.3.0 and there was no prompt to send or disable telemetry data.

Running with `GATSBY_TELEMETRY_DEBUG=1` shows a few Captured events.

**Comment by DSchau** [comment]
@millette were you using yarn?

[Yarn has a bug](https://github.com/yarnpkg/yarn/issues/5476) where a `postinstall` script isn't invoked.

[We've fixed the issue here](https://github.com/gatsbyjs/gatsby/pull/12867) so that all users see the prompt! We've published the fix in `gatsby-telemetry`@`1.0.1` which is required by `gatsby`@`2.3.1`.

Going to close this out, but thanks for the note and sorry about the little blip, here!

**Comment by millette** [comment]
@DSchau Thanks for the quick response. Yes, using yarn I tried again with gatsby 2.3.1, verified it upgraded telemetry to 1.0.1 and ran `gatsby build` but there still was no mention of telemetry.

Completely unrelated but perhaps I can also have your attention on #12755.

**Comment by millette** [comment]
Also tried 2.3.1, no telemetry prompt or message. Then I noticed the new file `~/.config/gatsby/config.json` and when I removed it, I finally saw the telemetry message.

Since a few people might have installed 2.3 without seeing a message, may I suggest a `confirmed` field in `config.json` and display the message if it's not set. This way, the message will appear for those who have missed it. Feel free to close (again) if you don't think this precaution is necessary.
